builds/misc/templates/image-linux-rust-jobs.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
 ################################################################################
     displayName: Linux Rust ${{ parameters.archName }}
     pool:
-      vmImage: 'ubuntu-16.04'
+      vmImage: ubuntu-16.04
```

Comment:
Did you mean to update this to 18.04?

Reply:
No, these images fail to build if you upgrade the agent to 18.04. So I put them back to 16.04, and we can tackle it later.